.env.local.example

```diff
@@ -1,2 +0,0 @@
-# Copy this file and rename it to `.env.local` to configure your own WCL API key. You can get your own API key from the bottom of the Warcraft Logs settings: https://www.warcraftlogs.com/accounts/changeuser
```

Comment:
Please revert removing this file, it's needed for other future contributors.

Reply:
Put it back in.